modules/stanford_icon_grid/js/stanford_icon_grid.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @file
+ * Fontawesome field widget.
+ */
+
+(function ($) {
+  Drupal.behaviors.stanfordFontawesome = {
+    attach: function (context, settings) {
+      $('.stanford-fontawesome-icon').once(function () {
```

Comment:
From: https://www.lullabot.com/articles/understanding-javascript-behaviors-in-drupal

```
The first time that Drupal.attachBehaviors() is called, the context variable contains the document object representing the DOM, but for the rest of the calls context will hold the affected piece of HTML. This is often overlooked by developers, leading to inefficient code that either causes tricky bugs or stresses the browser.
```